superagi/models/workflows/agent_workflow_step.py

```diff
@@ -30,9 +30,9 @@
     agent_workflow_id = Column(Integer)
     unique_id = Column(String)
     step_type = Column(String)  # TRIGGER, NORMAL
-    action_type = Column(String)  # TOOL, ITERATION_WORKFLOW, LLM
+    action_type = Column(String)  # TOOL, ITERATION_WORKFLOW, LLM, WAIT_STEP
     action_reference_id = Column(Integer)  # id of the action
-    next_steps = Column(JSONB) # edge_ref_id, response, step_id
+    next_steps = Column(JSONB)  # edge_ref_id, response, step_id
 
     def __repr__(self):
         """
```

Comment:
The action type 'WAIT_STEP' is hardcoded. Consider using a constant or enum type to avoid hardcoding and improve maintainability.